.kitchen.yml

```diff
@@ -63,7 +48,7 @@ suites:
     run_list:
       - recipe[osquery_spec::osquery_wrapper]
     driver:
-        require_chef_omnibus: 11.18.12-1
+        require_chef_omnibus: latest
```

Comment:
I don't think we want to use `latest` here, because we can't accept the Chef 15 license terms. We'll need to pin to the latest version of Chef 14.

Reply:
Pinned 14.13